util/Migrator/Migrator.csproj

```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="dbup-sqlserver" Version="4.4.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.6" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
```

Comment:
See my general comment on upgrading all ASP.NET to 5. I don't think we want mis-matches

Reply:
Taking this chance to upgrade to .NET Core 5.0 would be cool, but is not necessary. 3.1 has long term support for [until the end of 2022](https://versionsof.net/core/), and EF 5  is supported on .NET Core versions [3.0 or higher](https://docs.microsoft.com/en-us/ef/core/miscellaneous/platforms). 

The upgrade seems non-invasive. Just nod if you still want to go ahead and get that done here.

Reply:
I vote yes. Sounds like if we want to use EF 5 then we should go ahead and upgrade ASP.NET to 5 across the board, especially if there aren't any breaking changes or deprecated methods/types with any what we're using today.

Reply:
It shall be so